lib/rubocop/cop/util.rb

```diff
@@ -152,6 +152,22 @@ def range_with_surrounding_space(range, side = :both, with_newline = true)
         Parser::Source::Range.new(buffer, begin_pos, end_pos)
       end
 
+      def range_by_whole_lines(range, include_final_newline = false)
```

Comment:
The idea was kindof like with "visual" mode in Vim, this method expands the input range to include all of any lines it covers (optionally with the final newline per the arg).